src/main/java/com/kodilla/ecommercee/dto/Group_Entity.java

```diff
@@ -0,0 +1,31 @@
+package com.kodilla.ecommercee.dto;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+
+@Entity
+public class Group_Entity {
```

Comment:
Czy jest jakiś powód od odejścia od konwencji i dodania PostFixa _Entity ? Może coś co wskazuje że jest to grupa z produktami

Reply:
Nie moglam zainicjowac nazwy klasy jako Group ani jako Groups bo nie zapisywalo sie w bazie danych. Dlatego zastosowalam taka przykladowa nazwe